tests/js/utilities/timezone.js

```diff
@@ -0,0 +1,10 @@
+/** @format */
```

Comment:
Moved from `client/utils/test-utils.js`, similar to how we do it in WooPay, so we don't risk importing these test utilities in the real application